spec/mailers/registrations_mailer_spec.rb

```diff
@@ -15,7 +15,6 @@
     let(:registration) { FactoryBot.create(:registration, user: french_user, competition: competition_with_organizers) }
     let(:mail_new) { RegistrationsMailer.notify_registrant_of_new_registration(registration) }
     let(:mail_accepted) { RegistrationsMailer.notify_registrant_of_accepted_registration(registration) }
-    let(:mail_pending) { RegistrationsMailer.notify_registrant_of_pending_registration(registration) }
```

Comment:
Why is this disappearing without replacement? There is `notify_registrant_of_waitlisted_registration` now, which you could well put here.

Reply:
The header test seems to rely on a french translation key which we don't have - should I add it in in order for the test to pass? I can find some french people to consult with

Reply:
Nah, just a comment that we should add this as soon as French is translated should suffice

Reply:
On the other hand, French does get updated rather infrequently.
If you do indeed have contacts to French speakers (ideally, cubers) it would be great if you could add the relevant snippets.

Reply:
Asked for a translation from some French delegates :+1:

Reply:
And merge is resolved